dolphinscheduler-python/pydolphinscheduler/examples/task_datax_example.py

```diff
@@ -29,14 +29,67 @@
 from pydolphinscheduler.tasks.datax import CustomDataX, DataX
 
 # datax json template
-JSON_TEMPLATE = ""
+JSON_TEMPLATE = {
+    "job": {
+        "content": [
+            {
+                "reader": {
+                    "name": "mysqlreader",
+                    "parameter": {
+                        "username": "usr",
+                        "password": "pwd",
+                        "column": [
+                            "id",
+                            "name",
+                            "code",
+                            "description"
+                        ],
+                        "splitPk": "id",
+                        "connection": [
+                            {
+                                "table": [
+                                    "source_table"
+                                ],
+                                "jdbcUrl": [
+                                    "jdbc:mysql://127.0.0.1:3306/source_db"
+                                ]
+                            }
+                        ]
+                    }
+                },
+                "writer": {
+                    "name": "mysqlwriter",
+                    "parameter": {
+                        "writeMode": "insert",
+                        "username": "usr",
+                        "password": "pwd",
+                        "column": [
+                            "id",
+                            "name"
+                        ],
+                        "connection": [
+                            {
+                                "jdbcUrl": "jdbc:mysql://127.0.0.1:3306/target_db",
+                                "table": [
+                                    "target_table"
+                                ]
+                            }
+                        ]
+                    }
+                }
+            }
+        ]
```

Comment:
I failed to run this task. Error message:
![image](https://user-images.githubusercontent.com/8847400/148183682-e2b7eecb-e81e-46d6-82f4-55a247e7ab70.png)
So I think you should add `setting` key after `content` key.
```
 "setting":{
    "errorLimit":{
    	   "percentage":0,
    		"record":0
    	},
    	"speed":{
    		"channel":1,
    		"record":1000
    	}
 }
```

Reply:
Please confirm whether there is any problem with this. @zhongjiajie

Reply:
Ok, I would take a look

Reply:
> Please confirm whether there is any problem with this. @zhongjiajie

@devosend  I add the required attribute `setting.speed` for json define. see https://github.com/alibaba/DataX/blob/master/userGuid.md#quick-start , I also add some optional setting basic on your advice above

Reply:
But I have one more thing to said, I try to start datax jobs in my local environment but failed 😭 😭 😭  . So I can not reproduce the case you said above https://github.com/apache/dolphinscheduler/pull/7801#discussion_r778621989

Reply:
I run with web ui, t's correct. are there any error tips?

Reply:
Did you test this PR code? If you did and it could run datax task, make we could directly merge this PR now.

Reply:
> are there any error tips

My laptop is lack of resources to run full standalone server, it only could run api-server 😭

Reply:
I run it in my local. it's ok. Merge it please. @zhongjiajie

Reply:
Thank!